sdv/single_table/base.py

```diff
@@ -61,7 +61,11 @@ def __init__(self, metadata, enforce_min_max_values=True, enforce_rounding=True)
         self.metadata.validate()
         self.enforce_min_max_values = enforce_min_max_values
         self.enforce_rounding = enforce_rounding
-        self._data_processor = DataProcessor(metadata)
+        self._data_processor = DataProcessor(
+            metadata=metadata,
+            learn_rounding_scheme=enforce_rounding,
```

Comment:
minor: could we update the argument name in the `DataProcessor` to match (ie. `learn_rounding_scheme` -> `enforce_rounding`)